bin/stacks/routing-database-stack.ts

```diff
@@ -36,6 +36,11 @@ export const DynamoDBTableProps = {
     PartitionKeyName: 'cacheKey',
     SortKeyName: 'block',
   },
+  TokenValidationResultDbTable: {
```

Comment:
shall we maybe call it token proprties table? Given that we should be reusing this for other stuff in the future?

Reply:
Renamed